frontend/src/App.js

```diff
@@ -1,29 +1,31 @@
-import React from "react";
-import {
-  BrowserRouter as Router,
-  Route,
-  Routes
-
-} from "react-router-dom";
-
-
-import logo from './logo.svg';
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import './App.css';
-import Header from './components/Header'
-import RoomListPage from './pages/RoomListPage'
-import RoomPage from './pages/RoomPage'
+import RoomListPage from './pages/RoomListPage';
+import RoomPage from './pages/RoomPage';
+import Header from './components/Header';
+import Login from './components/Login';
+import Register from './components/Register';
+import CreateRoom from './components/CreateRoom';
+import { CookiesProvider } from 'react-cookie';
 function App() {
   return (
-    <Router>
-      <div className="container dark"> 
-        <div className="app">
-          <Routes>
-              <Route path="/" exact  element={<RoomListPage />} />
-              <Route path="/room/:id"  element={<RoomPage />} />
-          </Routes>
+    <CookiesProvider>
+      <Router>
+        <div className="container dark">
+          <div className="app">
+            <Header />
+            <Routes>
+              <Route path="/" exact element={<RoomListPage />} />
```

Comment:
you should create separate file for routes e.g.: in `src/routes/Routes.js`

pack all routes around `<>` fragment component and then just import them to `App` component